prow.sh

```diff
@@ -381,7 +381,7 @@ configvar CSI_SNAPSHOTTER_VERSION "$(default_csi_snapshotter_version)" "external
 # whether they can run with the current cluster provider, but until
 # they are, we filter them out by name. Like the other test selection
 # variables, this is again a space separated list of regular expressions.
-configvar CSI_PROW_E2E_SKIP 'Disruptive' "tests that need to be skipped"
+configvar CSI_PROW_E2E_SKIP 'Disruptive|\[Feature:SELinux\]' "tests that need to be skipped"
```

Comment:
I think the correct form would be `\[Disruptive\]|\[Feature:SELinux\]`.
Example: https://github.com/kubernetes/test-infra/blob/62bfb1841cef4df5170795067116a94a0a891f01/config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml#L44

Reply:
Fixed